support/cas-server-support-actions-core/src/main/java/org/apereo/cas/web/flow/login/InitialFlowSetupAction.java

```diff
@@ -91,7 +91,9 @@ protected Event doExecuteInternal(final RequestContext context) throws Throwable
     protected String configureWebflowForTicketGrantingTicket(final RequestContext context) {
         val request = WebUtils.getHttpServletRequestFromExternalWebflowContext(context);
         val ticketGrantingTicketId = ticketGrantingTicketCookieGenerator.retrieveCookieValue(request);
+        LOGGER.debug("Retrieved the TGT identifier in the login webflow: [{}]", ticketGrantingTicketId);
         val ticket = ticketRegistrySupport.getTicketGrantingTicket(ticketGrantingTicketId);
+        LOGGER.debug("Retrieved the ticket granting ticket in the login webflow: [{}]", ticket);
```

Comment:
You will need to be consistent: one area says TGT and the other says ticket granting ticket. It should likely be `ticket-granting ticket` and you should mark both under trace.

Reply:
Yes, I have updated both PRs.